config/locales/en.yml

```diff
@@ -89,6 +89,15 @@ en:
           msg: "There's no meeting currently running in this room"
           title: "Meeting not running"
     bigbluebutton: BigBlueButton
+    meetings:
+      delete:
+        success: "The meeting was successfully deleted."
+      notice:
+        destroy:
+          success_with_bbb_error: "The meeting was successfully destroyed but it wasn't deleted from the webconference server (\"%{error}\")"
```

Comment:
new message: " The register was successfully destroyed but it wasn't deleted from the videoconference"